docs/ann_types.md

```diff
@@ -46,88 +47,87 @@ Annotations with custom types:
+| Name in snpEff `ANN`       | Type                                                   | Description | Example expression                               |
+|----------------------------|--------------------------------------------------------|-------------|--------------------------------------------------|
+| `Annotation`               | `Consequences` / `list[Term]`                          |             | `'ANN["Annotation"].any_is_a("intron_variant")'` |
+| `cDNA.pos / cDNA.length`   | `PosRange` with properties `start`, `end` and `length` |             | `'ANN["cDNA.pos / cDNA.length"].start < 42'`     |
+| `CDS.pos / CDS.length`     | `PosRange` with properties `start`, `end` and `length` |             | `'ANN["CDS.pos / CDS.length"].end > 42'`         |
+| `AA.pos / AA.length`       | `PosRange` with properties `start`, `end` and `length` |             | `'ANN["AA.pos / AA.length"].length == 42'`       |
+| `ERRORS / WARNINGS / INFO` | `list[str]`                                            |             |                                                  |
 
 
 Annotations with type `str`:
 
-| Name in snpEff `ANN` | Type  | Description | Example expression |
-|----------------------|-------|-------------|--------------------|
-| `Allele`             | `str` |             |                    |
-| `Annotation`         | `str` |             |                    |
-| `Annotation_Impact`  | `str` |             |                    |
-| `Gene_Name`          | `str` |             |                    |
-| `Gene_ID`            | `str` |             |                    |
-| `Feature_Type`       | `str` |             |                    |
-| `Feature_ID`         | `str` |             |                    |
-| `Transcript_BioType` | `str` |             |                    |
-| `Rank`               | `str` |             |                    |
-| `HGVS.c`             | `str` |             |                    |
-| `HGVS.p`             | `str` |             |                    |
-| `Distance`           | `str` |             |                    |
+| Name in snpEff `ANN` | Description | Example expression |
+|----------------------|-------------|--------------------|
+| `Allele`             |             |                    |
+| `Annotation_Impact`  |             |                    |
+| `Gene_Name`          |             |                    |
+| `Gene_ID`            |             |                    |
+| `Feature_Type`       |             |                    |
+| `Feature_ID`         |             |                    |
+| `Transcript_BioType` |             |                    |
+| `Rank`               |             |                    |
+| `HGVS.c`             |             |                    |
+| `HGVS.p`             |             |                    |
+| `Distance`           |             |                    |
```

Comment:
Can we easily add descriptions, here? I think most of them can probably be copied from the vep table above?

Also, I'd delete the Example expression column, unless you want to add information in there.